src/app/services/wallet.service.ts

```diff
@@ -200,10 +208,17 @@ export class WalletService {
       .reduce((a , b) => a + b, 0));
   }
 
-  private addWallet(wallet) {
-    this.all.first().subscribe(wallets => {
-      wallets.push(wallet);
-      this.saveWallets(wallets);
+  transaction(txid: string): Observable<any> {
```

Comment:
This function does not seem to be used anywhere.

Reply:
Fixed